dataloader.opam

```
@@ -1,17 +1,18 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer: "Andreas Garnaes <andreas.garnaes@gmail.com>"
 authors: "Andreas Garnaes <andreas.garnaes@gmail.com>"
 homepage: "https://github.com/andreas/ocaml-dataloader"
 doc: "https://andreas.github.io/ocaml-dataloader/"
 bug-reports: "https://github.com/andreas/ocaml-dataloader/issues"
-dev-repo: "https://github.com/andreas/ocaml-dataloader.git"
-build: [["jbuilder" "build" "-p" name "-j" jobs]]
-build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+dev-repo: "git+https://github.com/andreas/ocaml-dataloader.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
 depends: [
-  "jbuilder" {build}
+  "ocaml" {>= "4.03.0"}
+  "dune" {build & >= "1.0"}
   "rresult"
-  "alcotest" {test}
-]
-available: [
-  ocaml-version >= "4.03.0"
+  "alcotest"
```

Same consideration as above.

```suggestion
  "alcotest" {with-test}
```